test/site/www/base-url/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
```

Comment:
@christian-bromann I retrieve this page in this test case https://github.com/webdriverio/webdriverio/pull/1946/files#diff-f445c8ec5b775aaf329a644cb15a09b8L8